SPECS/cyrus-sasl-ldap/cyrus-sasl-ldap.spec

```diff
@@ -0,0 +1,214 @@
+# Directory in which to place saslauthd's listening socket, pid file, and so
+# on.  This directory must already exist.
+SOCKETDIR=/run/saslauthd
+
+# Mechanism to use when checking passwords.  Run "saslauthd -v" to get a list
+# of which mechanism your installation was compiled with the ablity to use.
+MECH=pam
+
+# Additional flags to pass to saslauthd on the command line.  See saslauthd(8)
+# for the list of accepted flags.
+FLAGS=
+EOF
+
+mkdir -p %{buildroot}/lib/systemd/system
+cat << EOF >> %{buildroot}/lib/systemd/system/saslauthd.service
+[Unit]
+Description=SASL authentication daemon.
+
+[Service]
+Type=forking
+PIDFile=/run/saslauthd/saslauthd.pid
+EnvironmentFile=%{_sysconfdir}/sysconfig/saslauthd
+ExecStart=%{_sbindir}/saslauthd -m \$SOCKETDIR -a \$MECH \$FLAGS
+RuntimeDirectory=saslauthd
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+install -vdm755 %{buildroot}%{_libdir}/systemd/system-preset
+echo "disable saslauthd.service" > %{buildroot}%{_libdir}/systemd/system-preset/50-saslauthd.preset
+
+%check
+make %{?_smp_mflags} check
+
+%post
+%{_sbindir}/ldconfig
+
+%postun
+/sbin/ldconfig
+
+%files
+%defattr(-,root,root)
+%exclude %{_sysconfdir}/sysconfig/saslauthd
+%exclude /lib/systemd/system/saslauthd.service
+%exclude %{_libdir}/systemd/system-preset/50-saslauthd.preset
+%exclude %{_sbindir}/*
+%exclude %{_datadir}/licenses/%{name}/LICENSE
+%exclude %{_mandir}/man8/*
+
+# %files devel
+%exclude %{_includedir}/*
+%exclude %{_libdir}/libsasl2.so
+%exclude %{_libdir}/pkgconfig/*.pc
+%exclude %{_mandir}/man3/*
+
+# %files gs2
+%exclude %{_plugindir2}/libgs2.so
+%exclude %{_plugindir2}/libgs2.so.%{_soversion}*
+
+# %files gssapi
+%exclude %{_plugindir2}/libgssapiv2.so
+%exclude %{_plugindir2}/libgssapiv2.so.%{_soversion}*
+
+# %files ldap
+%{_plugindir2}/libldapdb.so
+%{_plugindir2}/libldapdb.so.%{_soversion}*
+
+# %files lib
+#%exclude COPYING
+#%exclude doc/html/*.html
+%exclude %{_libdir}/libsasl2.so.%{_soversion}*
+%exclude %dir %{_plugindir2}/
+%exclude %{_plugindir2}/libanonymous.so
+%exclude %{_plugindir2}/libanonymous.so.%{_soversion}*
+%exclude %{_plugindir2}/libsasldb.so
+%exclude %{_plugindir2}/libsasldb.so.%{_soversion}*
+%exclude %{_sbindir}/saslpasswd2
+%exclude %{_sbindir}/sasldblistusers2
+
+# %files md5
+%exclude %{_plugindir2}/libcrammd5.so
+%exclude %{_plugindir2}/libcrammd5.so.%{_soversion}*
+%exclude %{_plugindir2}/libdigestmd5.so
+%exclude %{_plugindir2}/libdigestmd5.so.%{_soversion}*
+
+# %files ntlm
+%exclude %{_plugindir2}/libntlm.so
+%exclude %{_plugindir2}/libntlm.so.%{_soversion}*
+
+# %files plain
+%exclude %{_plugindir2}/liblogin.so
+%exclude %{_plugindir2}/liblogin.so.%{_soversion}*
+%exclude %{_plugindir2}/libplain.so
+%exclude %{_plugindir2}/libplain.so.%{_soversion}*
+
+# %files scram
+%exclude %{_plugindir2}/libscram.so
+%exclude %{_plugindir2}/libscram.so.%{_soversion}*
+
+# %files sql
+%exclude %{_plugindir2}/libsql.so
+%exclude %{_plugindir2}/libsql.so.%{_soversion}*
```

Comment:
I'd only re-order these to make it easier to see what actually goes into this package and add `-n` to avoid confusion:
```suggestion
%defattr(-,root, root)
%{_plugindir2}/libldapdb.so
%{_plugindir2}/libldapdb.so.%{_soversion}*

# %files -n %{_base_name}
%exclude %{_sysconfdir}/sysconfig/saslauthd
%exclude /lib/systemd/system/saslauthd.service
%exclude %{_libdir}/systemd/system-preset/50-saslauthd.preset
%exclude %{_sbindir}/*
%exclude %{_datadir}/licenses/%{name}/LICENSE
%exclude %{_mandir}/man8/*

# %files -n %{_base_name}-devel
%exclude %{_includedir}/*
%exclude %{_libdir}/libsasl2.so
%exclude %{_libdir}/pkgconfig/*.pc
%exclude %{_mandir}/man3/*

# %files -n %{_base_name}-gs2
%exclude %{_plugindir2}/libgs2.so
%exclude %{_plugindir2}/libgs2.so.%{_soversion}*

# %files -n %{_base_name}-gssapi
%exclude %{_plugindir2}/libgssapiv2.so
%exclude %{_plugindir2}/libgssapiv2.so.%{_soversion}*

# %files -n %{_base_name}-lib
#%exclude COPYING
#%exclude doc/html/*.html
%exclude %{_libdir}/libsasl2.so.%{_soversion}*
%exclude %dir %{_plugindir2}/
%exclude %{_plugindir2}/libanonymous.so
%exclude %{_plugindir2}/libanonymous.so.%{_soversion}*
%exclude %{_plugindir2}/libsasldb.so
%exclude %{_plugindir2}/libsasldb.so.%{_soversion}*
%exclude %{_sbindir}/saslpasswd2
%exclude %{_sbindir}/sasldblistusers2

# %files -n %{_base_name}-md5
%exclude %{_plugindir2}/libcrammd5.so
%exclude %{_plugindir2}/libcrammd5.so.%{_soversion}*
%exclude %{_plugindir2}/libdigestmd5.so
%exclude %{_plugindir2}/libdigestmd5.so.%{_soversion}*

# %files -n %{_base_name}-ntlm
%exclude %{_plugindir2}/libntlm.so
%exclude %{_plugindir2}/libntlm.so.%{_soversion}*

# %files -n %{_base_name}-plain
%exclude %{_plugindir2}/liblogin.so
%exclude %{_plugindir2}/liblogin.so.%{_soversion}*
%exclude %{_plugindir2}/libplain.so
%exclude %{_plugindir2}/libplain.so.%{_soversion}*

# %files -n %{_base_name}-scram
%exclude %{_plugindir2}/libscram.so
%exclude %{_plugindir2}/libscram.so.%{_soversion}*

# %files -n %{_base_name}-sql
%exclude %{_plugindir2}/libsql.so
%exclude %{_plugindir2}/libsql.so.%{_soversion}*
```